README.md

```diff
@@ -121,20 +121,19 @@ const MONACO_DIR = path.resolve(__dirname, './node_modules/monaco-editor');
 
 ## Properties
 
-If you specify `value` property, the component behaves in controlled mode.
-Otherwise, it behaves in uncontrolled mode.
+All the properties below are optional.
 
 - `width` width of editor. Defaults to `100%`.
 - `height` height of editor. Defaults to `100%`.
-- `value` value of the auto created model in the editor.
+- `value` value of the auto created model in the editor. If specified, then the component behaves in controlled mode. Otherwise, it behaves in uncontrolled mode.
```

Comment:
What is controlled mode?

Reply:
I don't know, bit it was earlier written above all the properties, rather than next to the one it's related to, so I moved it also for better readability.

What it means I don't know. I tried to set it, but it didn't lock the editor to the given value.

Reply:
Let's delete it :-P

Reply:
Haha, done.